### PR TITLE
Add Kiwi For G Suite v2.0.18v

### PR DESCRIPTION
**Pull request by LanikSJ**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by LanikSJ** [comment]
I could use some help with:
```bash
± |kiwi-for-gsuite ✓| → brew cask reinstall kiwi-for-g-suite
==> Satisfying dependencies
==> Downloading https://s3.amazonaws.com/kiwiforgsuite/mac/release/Kiwi+for+G+Suite.pkg
Already downloaded: /Users/ierenstein/Library/Caches/Homebrew/downloads/f363af0fcffb68e50eaad3f4c35653f645cd6b572b6d4000f3bb69f33389f04f--Kiwi for G Suite.pkg
==> Verifying SHA-256 checksum for Cask 'kiwi-for-g-suite'.
==> Installing Cask kiwi-for-g-suite
==> Purging files for version 2.0.18v of Cask kiwi-for-g-suite
Error: It seems the App source '/usr/local/Caskroom/kiwi-for-g-suite/2.0.18v/Kiwi for G Suite.app' is not there.
```
I'm a noob at submitting casks so go easy on me. 🙂

File: Casks/kiwi-for-g-suite.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '2.0.18v'
+  sha256 'e7f87a89902b009504b311d07ede20b7613d16acb4d39d4772e229e0d3a41cf0'
+
+  # s3.amazonaws.com/kiwiforgsuite/mac/release/Kiwi+for+G+Suite.pkg was verified as official when first introduced to the cask
+  url 'https://s3.amazonaws.com/kiwiforgsuite/mac/release/Kiwi+for+G+Suite.pkg'
+  appcast 'https://rink.hockeyapp.net/api/2/apps/0e823ae8232540248fd3980052c56495/'
+  name 'Kiwi For G Suite'
+  homepage 'https://www.kiwiforgmail.com'
+
+  app 'Kiwi for G Suite.app'
```
Comment (vitorgalvao) [inline Casks/kiwi-for-g-suite.rb]: You don’t use `app`, here, you use [`pkg`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/pkg.md) and [`uninstall`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md).

**Review by vitorgalvao — APPROVED** [review]

File: Casks/kiwi-for-g-suite.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/kiwi-for-g-suite.rb]: You don’t use `app`, here, you use [`pkg`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/pkg.md) and [`uninstall`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/kiwi-for-g-suite.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'kiwi-for-g-suite' do
+  version '2.0.18v'
+  sha256 'e7f87a89902b009504b311d07ede20b7613d16acb4d39d4772e229e0d3a41cf0'
+
+  # s3.amazonaws.com/kiwiforgsuite/mac/release/Kiwi+for+G+Suite.pkg was verified as official when first introduced to the cask
+  url 'https://s3.amazonaws.com/kiwiforgsuite/mac/release/Kiwi+for+G+Suite.pkg'
```
Comment (vitorgalvao) [inline Casks/kiwi-for-g-suite.rb]: I only see a way to download from the Mac App Store, where are you getting this `url` from?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/kiwi-for-g-suite.rb (hunk 2 above)
Comment (LanikSJ) [reply Casks/kiwi-for-g-suite.rb]: This URL came from their website: https://www.kiwiforgmail.com/business.php?tab=benefits towards the bottom.

File: Casks/kiwi-for-g-suite.rb (hunk 1 above)
Comment (LanikSJ) [reply Casks/kiwi-for-g-suite.rb]: No problem I looked up the FoxIt Reader submission for ideas how to do the uninstall.  Not sure if both entries are required so if they're not I'll gladly remove one of them.

File: Casks/kiwi-for-g-suite.rb (hunk 2 above)
Comment (LanikSJ) [reply Casks/kiwi-for-g-suite.rb]: Also the App Store version is Kiwi For Gmail vs this is Kiwi for G Suite their Enterprise version.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
All good. Thank you.

**Pull request merged**